.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+jdk:
+  - openjdk8
```

Comment:
Will this be used for all downstream projects by default?

Reply:
No, the travis files only apply to the repo they are in, nothing else.